ui/scripts/vpc.js

```diff
@@ -745,12 +745,24 @@
                         }
                     },
                     dataProvider: function(args) {
+                        var array1 = [];
+                        if (args.filterBy != null) {
+                            if (args.filterBy.search != null && args.filterBy.search.by != null && args.filterBy.search.value != null) {
+                                if (args.filterBy.search.by === "name") {
+                                    if (args.filterBy.search.value.length > 0)
+                                        array1.push("&keyword=" + args.filterBy.search.value);
+                                }
+                            }
+                        }
+                        var data = {
+                            page: args.page,
+                            pageSize: pageSize,
+                            networkid: args.context.networks[0].id,
+                            listAll: true
+                        };
                         $.ajax({
-                            url: createURL('listLoadBalancers'),
-                            data: {
-                                networkid: args.context.networks[0].id,
-                                listAll: true
-                            },
+                            url: createURL('listLoadBalancers' + array1.join("")),
+                            data: data,
                             success: function(json) {
```

Comment:
@rhtyd Changed the code as mentioned.

Reply:
@rhtyd existing code is taking array of keywords but your implementation can take only one keyword.

Reply:
@vedulasantosh the code I see here only pushed a single key/value in the array, furthermore the array is joined using blank string:
`                                        array1.push("&keyword=" + args.filterBy.search.value);